Makefile

```diff
@@ -132,6 +132,7 @@ build:
 
 image: build build-push-image
 
+# as an alternative, you can also config docker in your travis.yaml
```

Comment:
Can you help add a link of example for reference? @morvencao

Reply:
I can't find an example.

Reply:
If cannot find an example link, can you append an example `travis.yml` in the comment?